lib/ood_core/job/adapters/slurm.rb

```diff
@@ -369,7 +440,7 @@ def status(id)
           id = id.to_s
           jobs = @slurm.get_jobs(
             id: id,
-            filters: [:job_id, :array_job_task_id, :state_compact]
+            attrs: [:job_id, :array_job_task_id, :state_compact]
```

Comment:
This is a good decision. I prefer that we use `filters` for when we are attempting to filter which records are returned rather than which record attributes are returned.